specification/glossary.md

```diff
@@ -161,3 +164,25 @@ Pseudorandomly resolve flag values using a context property, such as a targeting
 ### Rule
 
 A rule is some criteria that's used to determine which variant a particular context should be mapped to.
+
+## SDK Paradigms
+
+Feature flag frameworks tend to come in two categories: those designed for use with a single user client application, and those designed for multi-user applications, such as web server applications. Some parts of the OpenFeature specification diverge depending on which paradigm the implementation seeks to adhere to.
```

Comment:
I wouldn't say that the frameworks come in two categories - many frameworks support both paradigms. I'd say "have SDKs which operate in two distinct paradigms" instead.

Reply:
Updated.